packages/mapviewer/src/css/tailwind.css

```diff
@@ -0,0 +1,2 @@
+@import 'tailwindcss' prefix(tw);
+@import '@geoadmin/tooltip/tailwindcss';
```

Comment:
Is this line doing anything?

Reply:
without it, the tailwind classes from the `@geoadmin/tooltip` module are ignored and not taken into account in the `mapviewer` package's tailwind output